docs/ipycytoscape_example.rst

```diff
@@ -0,0 +1,137 @@
+====================
+IPyCytoscape Example
+====================
+
+Thebe can display output from ipycytoscape_, which could visualize graphs using Cytoscape.js_.
+
+.. _ipycytoscape: https://github.com/QuantStack/ipycytoscape 
+.. _Cytoscape.js: https://js.cytoscape.org/ 
+
+Setup
+=====
+
+Be sure to load require.js before any of your thebe activation code so that the
+Cytoscape visualizations can function:
+
+.. code-block:: html
+
+   <script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.4/require.min.js"></script>

```

Comment:
Did you verify that this is required? (no pun intended)

Reply:
Yes, this is required, or the widget fails to load.
![image](https://user-images.githubusercontent.com/22624020/97248511-32003980-17bf-11eb-923f-0987b9a166b9.png)

This is from the same code except removing require.js.

Reply:
Thanks. It would be nice to know which ones require require.js. Seems like all of the ones we've tested so far.